assets/elm/main.elm

```diff
@@ -1,11 +1,9 @@
 module Main exposing (..)
 
---where
-
 import Html exposing (Html, br, button, div, form, h3, input, li, table, tbody, td, text, tr, ul)
 import Html.Events exposing (onClick, onInput, onSubmit)
-import Json.Decode as JD exposing (field)
-import Json.Encode as JE
```

Comment:
My thoughts: I'd like to avoid aliases for now, since we're rookies to all the libraries. I think it'll be more confusing for me this early on.